### ISSUE TITLE
Revise card states

### ISSUE THREAD

**Issue by stefandesu** [gbv/cocoda#371]
Currently, cards can have two states: open or minimized. We should add a third state "hidden" that completely hides a card. On top of that, we could add a "full screen" button to cards that hide all other cards.

We need to figure out:

- how to best integrate this into the UI
- how to unhide cards if they are completely hidden
- possibly other problems with this

**Comment by nichtich** [comment]
* mapping editor should always be hidden if no concepts are selected
* add a "fullscreen" button to the mapping browser to hide mapping editor and left/right concept browser

**Comment by stefandesu** [comment]
> * mapping editor should always be hidden if no concepts are selected

It should not be hidden if a mapping is edited, even when no concepts are selected. I think it should also be shown if only schemes are selected even when no concepts are selected (you could still create mappings by dragging them into Mapping Editor), but I'm not dead set on this.

**Comment by nichtich** [comment]
If both mapping editor and mapping browser are minimized, they should not take much horizontal space, should they? The interface could be used to compare two concept schemes without mappings in the center.

**Comment by stefandesu** [comment]
I agree. But before we make another half-baked implementation, I think we should solve this properly. There are already some questions in the previous comments here and I'm sure there will be more as we make this more concrete.

**Comment by stefandesu** [comment]
This requires further discussion, so I'll move it to the next milestone.
